pro-planner-client/src/components/UserList.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import {Button, Form} from "react-bootstrap";
+import {useSelector, useDispatch} from 'react-redux';
+import {selectUser, navigateToCreationPage} from '../redux/userSlice';
+
+const UserList = () => {
+    const userList = useSelector((state) => state.user.userList);
+    const selectedUser = useSelector((state) => state.user.selectedUser);
+    const dispatch = useDispatch();
+
+
+    if (!userList || userList.length === 0) {
+        return <p>No User Yet</p>
+    }
+
+    const handleUserSelect = (user) => {
+        dispatch(selectUser(user));
+    };
+
+    const handleContinueClick = () => {
+        if (selectedUser) {
+            dispatch(navigateToCreationPage());
```

Comment:
Navigation to another page wouldn't be done through dispatching actions to redux. This would be done through react-router. For a sample of what that looks like, the landing-page PR's latest commit should have logic for that.